specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -11011,6 +11360,17 @@
     },
     "VirtualMachineScaleSetNetworkProfile": {
       "properties": {
+        "networkApiVersion": {
```

Comment:
Need to add deleteOption for the publicIpadress config properties (Update And Create)